### DISCUSSION TITLE
Add Boolean Type

### DISCUSSION THREAD

**Discussion by ant-04** [community/community#5849] [Projects and Issues]
**Current Behaviour**
There are currently only 4 types when creating a new field (column)
![image](https://user-images.githubusercontent.com/65191196/135005257-7b166631-95a8-48f6-8cbb-81e0f96449d2.png)

**Expected behaviour**
Add support for boolean type. This is useful for checklist type tasks.

**Comment by Katsute** [comment]
You can use a single select with true/false as options.

**Reply by Katsute** [reply]
Alternatively you can also use ✔️/❌
